docker/Dockerfile.viewer

```diff
@@ -112,7 +112,7 @@ RUN pip install --no-cache-dir -r /tmp/requirements.txt
 RUN c_rehash
 
 # QT Base from packages does not support eglfs
-ARG PI_VERSION=pi3
+ARG PI_VERSION
```

Comment:
Let's leave this as a default. We have ways to work around this in `docker-compose.override.yml. for Docker on Buster, and [a different way](https://www.balena.io/docs/reference/balena-cli/#-b---buildarg-buildarg) to work around this for balena for non-Pi 3.